### DISCUSSION TITLE
Bookmarklet with credentials?

### DISCUSSION THREAD

**Discussion by lantura** [mealie-recipes/mealie#1217] [Feature Request]
You have to enter the credentials afterwards when using the bookmarklet. Is it possible to integrate them into the bookmarklet so that the login happens automatically?

This is now supported and documented https://hay-kot.github.io/mealie/documentation/recipes/recipes/#using-bookmarklets

I won't be making any extensions, but it's a cool idea! It could even be flexible enough to plugin to other self-hosted options in the recipe space. If anyone puts something together, let me know and I can add a note in the documentation

_Originally posted by @hay-kot in https://github.com/hay-kot/mealie/discussions/318#discussioncomment-1140068_

**Comment by 97WaterPolo** [comment]
+1
Would love to see this feature or some implementation of auto-logging user in VIA url. I've tried using JS/JQuery to open up login and enter credentials but can't easily support it on different platforms.

**Comment by hay-kot** [comment]
1) There are directions on submitting a feature requests

- https://github.com/hay-kot/mealie/issues/317

2) This isn't likely something I'm going to work on. I'm much more interested in building a browser extension that would interact with the API.

**Comment by lantura** [comment]
Hi,

the link
<https://hay-kot.github.io/mealie/documentation/recipes/recipes/#using-bookmarklets>in
the ticket does not work.

Thanks ...


Am 21.08.2024 um 18:18 schrieb Michael Genson:

**Reply by Kuchenpirat** [reply]
This would be the new link: 
https://docs.mealie.io/documentation/community-guide/import-recipe-bookmarklet/
